hooks/pre/30-upgrade.rb

```diff
@@ -102,6 +102,13 @@ def mark_qpid_cert_for_update
   end
 end
 
+def delete_event_queue
+  # piping output to /dev/null because on EL6 any qpid-stat/config commands gives a non fatal python traceback.
+  cert = '/etc/pki/katello/qpid_client_striped.crt'
+  Kafo::Helpers.execute("qpid-config --ssl-certificate #{cert} -b 'amqps://localhost:5671' del exchange event --durable > /dev/null 2>&1")
```

Comment:
The qpid-config help seems to imply `--durable`  is only valid of creating a queue